official/resnet/keras/keras_imagenet_benchmark.py

```diff
@@ -768,8 +761,8 @@ def benchmark_xla_8_gpu_fp16_tweaked_optional_next(self):
         'benchmark_xla_8_gpu_fp16_tweaked_optional_next')
     FLAGS.batch_size = 256 * 8  # 8 GPUs
     FLAGS.use_tensor_lr = True
-    # FLAGS.tf_gpu_thread_mode = 'gpu_private'
-    FLAGS.data_delay_prefetch = True
+    FLAGS.tf_gpu_thread_mode = 'gpu_private'
+    FLAGS.datasets_num_private_threads = 48
```

Comment:
hm, why do some of these tests need the threads setting explicitly but others don't?

Reply:
When we enable `gpu_private` thread mode, it automatically sets `datasets_num_private_threads` to (num of CPU cores - 3 x num of GPUs)---for each GPU, 2 private launching threads and 1 event listener thread. However we actually need 48 cores for data pipeline to saturate our model. Setting them to introduce thread contention on 64 vCPUs.

Reply:
For some tests (like the MLPerf-like one and `_delay_measure` ones), they target the 96 vCPU platform and we leave enough cores for data threads there.